loqusdb/build_models/case.py

```diff
@@ -93,7 +92,11 @@ def build_case(
             individual = case.individuals[ind_id]
             try:
                 # If a profile dict exists, get the profile for ind_id
-                profile = profiles[ind_id] if profiles else None
+                profile = profiles.get(ind_id) if profiles else None
```

Comment:
Would be great to start adding type hints

Reply:
It's a slippery slope... Before you know it they are everywhere. I agree though: reading carefully here, I don't understand how the previous couple of lines ever worked - and that would likely have been avoided with slightly stronger typing.